Emily_Practise.md

```diff
@@ -0,0 +1,9 @@
+# Emily Practise
+
+###Thing to do for co-op
+- Sienna minivan
+- github
+- Jenkins
+- perforce
```

Comment:
@Emily0731 the address is below :spider_web: http://confluence.adtran.com/display/EnRelEng/Perforce+Training+Course